### ISSUE TITLE
Logging in as MUD admin doesn't work

### ISSUE THREAD

**Issue by rgautier** [Seifert69/DikuMUD3#13]
Sorry, 
New issue with last night's build.  I can log in as a normal user, but if I log in as MUD admin, I can't perform the selection to enter MUD (or set Wizinvis, or exit):  

Output below:

By what name do they call you?
Papi
Welcome back Papi, please enter your password:
Welcome back Papi, you last visited Valhalla Test Mud on Sat Aug 1 19:39:11 2020
Message of Today
Located in: ~/etc/motd

90m/164e/35h>
Welcome to Valhalla

1) Enter Valhalla
W) Change Wizinv level [0]
0) Exit Valhalla

Make your choice:
1

1

1

1

1

It may be interesting to note that Papi was in the process of dying when I last quit (didn't eat/drink) and was at the part of the death process where you see clouds/the light, whatever, when I typed 'quit' to end my session.

**Comment by Seifert69** [comment]
Did you try to pull the latest?

I had been too eager on sanity checking while bug hunting. I already fixed
those issues half a day after.

Best Michael

**Comment by rgautier** [comment]
Yes, just pulled latest and recompiled again to be sure, same issue:
By what name do they call you?
Papi
Welcome back Papi, please enter your password:

Welcome back Papi, you last visited Valhalla Test Mud on Sat Aug 1 19:39:11 2020

Message of Today

Located in: ~/etc/motd



90m/164e/35h>
Welcome to Valhalla

1) Enter Valhalla
W) Change Wizinv level [0]
0) Exit Valhalla

Make your choice:
1
1
1

**Comment by Seifert69** [comment]
Can you mail me a zip of papi to michael@corleone.com? Thanks.

**Comment by rgautier** [comment]
I'll get you the file.

**Comment by Seifert69** [comment]
Found it. :-)

Solution was very simple. The DIL program that controls the menu just
needed to have top priority. It got blocked by the death_sequence.

While figuring it out, I stumbled on some other issues I had to fix.

You can pull & you can login with your admin. Oddly enough, you're dead and
will be taken to Valhalla :-) But everything works.

You can use the "wstat papi func" to see which functions are on your
char.use "zap papi stomach@update" to kill a DIL program.

You can essentially kill them all, quit entirely and login again. Then you
should be setup with the right DILs.

**Comment by rgautier** [comment]
No dice :(  Just pulled and recompiled fresh....

By what name do they call you?
Papi
Welcome back Papi, please enter your password:

Welcome back Papi, you last visited Valhalla Test Mud on Sat Aug 1 19:39:11 2020

Message of Today

Located in: ~/etc/motd



90m/164e/35h>
Welcome to Valhalla

1) Enter Valhalla
W) Change Wizinv level [0]
0) Exit Valhalla

Make your choice:
w
0
1

<no response>

**Comment by Seifert69** [comment]
Sorry you need to recompile the zones too. The bug was in the DIL code.

**Comment by rgautier** [comment]
I recompiled the zones, was able to log him in -- but death sequence never completed...I ended up stuck in Valhalla -- even after going down, west/east - it never drops me at the fountain.  (Not sure if this is intentional - usually it send dead newbies back to the newbie sign).

Anyway - THIS bug fixed...

**Comment by Seifert69** [comment]
I'm not sure either. But since you.somehow died with a god character I
think ill let it rest :-D
